socrates/lib/site/index.js

```diff
@@ -54,6 +54,10 @@ app.get('/history.html', function (req, res) {
   res.render('history');
 });
 
+app.get('/format.html', function (req, res) {
```

Comment:
As already written. I'd not change the name of the page.

btw. you forgot already to remove the old entry here. So links to the old entry give an error 500 :)

So just reuse the schedulte.html and render format there